src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1127,6 +1127,8 @@ namespace NServiceBus.Config
         [System.Configuration.ConfigurationPropertyAttribute("Threshold", DefaultValue="Info", IsRequired=true)]
         public string Threshold { get; set; }
     }
+    [System.ObsoleteAttribute("Please use `Switch to the code API by using \'EndpointConfiguration.EnlistWithLega" +
```

Comment:
We usually do not use `Please`

You can just write

`Switch to the code API`...

or `Use EndpointConfiguration.EnlistWithLega...` instead

Also instead will be added by the fody addin